AtmosphereAutopilot/Modules/AutopilotModule.cs

```diff
@@ -125,7 +125,7 @@ public virtual void Serialize()
         {
             BeforeSerialized();
             AutoSerialization.Serialize(this, module_name.Replace(' ', '_'),
-                KSPUtil.ApplicationRootPath + "GameData/AtmosphereAutopilot/designs/" + vessel.vesselName + ".txt",
+                KSPUtil.ApplicationRootPath + "GameData/AtmosphereAutopilot/designs/" + KSPUtil.SanitizeFilename(vessel.vesselName) + ".txt",
```

Comment:
Do the same on L107, please, in the method
public bool DeserializeVesselSpecific()

Reply:
Good catch, done. :+1: